4_transactions/send_batch.js

```diff
@@ -72,7 +72,8 @@ const main = async () => {
         if (currentHeight > startHeight) {
             break
         }
-        sleep(1)
+        console.log('Waiting for next block');
+        await sleep(1)
```

Comment:
`sleep(100)` will probably be more CPU friendly. :-)

Reply:
its converted to seconds but thanks